packages/graph/hash_graph/lib/graph/src/api/rest/property_type.rs

```diff
@@ -147,15 +152,10 @@ async fn get_property_type<P: StorePool + Send>(
     uri: Path<VersionedUri>,
     pool: Extension<Arc<P>>,
 ) -> Result<Json<PersistedPropertyType>, StatusCode> {
-    read_from_store(
-        pool.as_ref(),
-        &PropertyTypeQuery::new()
-            .by_uri(uri.base_uri())
-            .by_version(uri.version()),
-    )
-    .await
-    .and_then(|mut property_types| property_types.pop().ok_or(StatusCode::NOT_FOUND))
-    .map(Json)
+    read_from_store(pool.as_ref(), &Expression::for_versioned_uri(&uri.0))
+        .await
+        .and_then(|mut data_types| data_types.pop().ok_or(StatusCode::NOT_FOUND))
```

Comment:
These are property_types (untested)
```suggestion
        .and_then(|mut property_types| property_types.pop().ok_or(StatusCode::NOT_FOUND))
```

Reply:
Ah, missed that, thank you!